src/components/Chart/model.ts

```diff
@@ -90,6 +90,7 @@ export interface ReferenceArea<XDomain> {
   tickColor?: string
   stroke?: boolean
   strokeColor?: string
+  dashed?: boolean
```

Comment:
```suggestion
  strokeType?: 'line' | 'dashed'
```
It's more explicit and scalable